### ISSUE TITLE
Add Portfolio Profit/Loss Tracking

### ISSUE THREAD

**Issue by severmanolescu** [severmanolescu/Crypto-Articles-Bots#4]
What it does:

- > Tracks portfolio value over time.
- > Sends daily profit/loss updates.

How to Implement:

> Modify CryptoValue.py to store past values.
> Add percentage change calculations:

```
def calculate_profit_loss(initial_value, current_value):
    return ((current_value - initial_value) / initial_value) * 100
Add a new Telegram command /portfolio:
```

```
async def portfolio(update: Update, context: ContextTypes.DEFAULT_TYPE):
    profit_loss = calculate_profit_loss(initial_value, current_value)
    message = f"📊 Portfolio Value: ${current_value:.2f} \n🔻 Profit/Loss: {profit_loss:.2f}%"
    await update.message.reply_text(message)

```
Value:

> Monitors profit/loss in real time.
> Alerts when portfolio drops too much.
